src/types/proposals.ts

```diff
@@ -728,15 +728,25 @@ export type ProposalCommentsInDiscourse = {
   comments: ProposalComment[]
 }
 
-type VestingContractData = {
-  vestedAmount: number
+export type VestingContractData = {
+  vested_amount: number
   releasable: number
   released: number
   start_at: number
   finish_at: number
   vesting_total_amount: number
 }
 
+export type ProjectVestingData = {
+  status: ProjectStatus
```

Comment:
Why is the project status in the project vesting data? 
Maybe this property should be kept on ProposalProject